### PR TITLE
notebook for tiny sd

### PR DESCRIPTION
**Pull request by eaidova**
ref. CVS-117156

TO DO:

- [x] redraw plot
- [x] switch unet to static shapes (in next prerelease migration, current has issue with that)

### CODE REVIEW

File: notebooks/250-tiny-sd-image-generation/README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+![text2img.png](https://user-images.githubusercontent.com/29454499/260905732-f291d316-8835-4872-8d9b-8a1214448bfd.png)
+
+**Image-to-Image Generation Example**
+![img2img.png](https://user-images.githubusercontent.com/29454499/260905907-4b7835c6-1f63-4d00-a1ec-ccc4d7fca182.png)
+
+
+
+## Installation Instructions
+
+If you have not installed all required dependencies, follow the [Installation Guide](../../README.md).
```
[the remark this answers is not in the document]
Comment (eaidova) [reply notebooks/250-tiny-sd-image-generation/README.md:54]: not yet, here no torch installation inside

[the remark this answers is not in the document]
Comment (eaidova) [reply notebooks/250-tiny-sd-image-generation/README.md]: now, added

**Comment by igor-davidyuk** [comment]
> Could we import this code from a python module?
I meant - can we remove this code from the notebook and move it to a separate Python module so we can just import it?

**Comment by eaidova** [comment]
> > Could we import this code from a python module?
> 
> I meant - can we remove this code from the notebook and move it to a separate Python module so we can just import it?

no, it is not good practice, for documentation purposes it is recommended to keep all code in notebook (otherwise reading docs page user has no understanding where to find it and how it is looks like) and we are also lost education value without opportunity to explain this code, notebook just turn to the demo

**Review by igor-davidyuk — APPROVED** [review]
looks good

**Pull request merged**